<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallViewFrame</class>
 <widget class="QFrame" name="CallViewFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <widget class="QFrame" name="frame_callee">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_callee">
          <property name="text">
           <string>sip:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_codec">
          <property name="text">
           <string>G729</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_duration">
       <property name="title">
        <string>duration</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_duration">
          <property name="text">
           <string>00:00:00</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_answer">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Answer</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_decline">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Decline</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_mute">
         <property name="text">
          <string>Mute</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pause">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
